pkgs/applications/science/math/eukleides/default.nix

```diff
@@ -34,4 +65,4 @@ stdenv.mkDerivation {
     platforms = lib.platforms.linux;
```

Comment:
It turns out we can build this on Darwin with a minimal patch (replace gcc with clang), but in the interest of fixing the Hydra failure, let's not do this here. I'll send another PR after this is merged.

Reply:
Ok!